File: cp_rust/src/bin/abc432-a-permute-to-maximize.rs
// Created by Ayush Biswas at 2026/07/08 00:52
// https://atcoder.jp/contests/adt_easy_20260107_3/tasks/abc432_a
use cp_lib::*;

// @code begin
use cpio::*;

sol! {
    fn solution(
        (mut digs): [String]
    ) -> String {
        digs.sort();
        digs.reverse();
        digs.join("")
    }
}
// @code end

File: cp_nim/src/bin/abc432-a-permute-to-maximize.nim
# Created by Ayush Biswas at 2026/07/18 19:32
# https://atcoder.jp/contests/adt_easy_20260107_3/tasks/abc432_a
import ../lib/cpio

# @code begin

import algorithm, sequtils

var r = newReader()

let a = r[seq[int]].sorted.reversed

echo a.mapIt($it).join("")

# @code end
